src/Tribe/Repository.php

```diff
@@ -295,6 +298,10 @@
 	 * @var array A map of schema slugs and their meta keys to be queried.
 	 */
 	protected $simple_meta_schema = array();
+	/**
+	 * @var array A map of schema slugs and their taxonomies to be queried.
+	 */
+	protected $simple_tax_schema = array();
 	/**
```

Comment:
add a blank line before

Reply:
Sorry, was following the formatting that was there, lots of things without blank lines in here